lib/http.c

```diff
@@ -90,6 +90,11 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+/* Authentication mechanisms that do not transfer clear password. */
+#define SAFE_AUTH_MECHS (CURLAUTH_NEGOTIATE | CURLAUTH_BEARER |         \
+                         CURLAUTH_DIGEST | CURLAUTH_NTLM |              \
+                         CURLAUTH_NTLM_WB | CURLAUTH_AWS_SIGV4)
+
```

Comment:
Shouldn't this be the same bitmask as `CURLAUTH_ANYSAFE` ?

Reply:
Yes, I missed it. Will change.